src/IssueViz.Security.UnitTests/Taint/TaintStoreTests.cs

```diff
@@ -60,7 +60,7 @@ public void Set_NullCollection_ArgumentNullException()
         {
             var testSubject = CreateTestSubject();
 
-            Action act = () => testSubject.Set(null);
+            Action act = () => testSubject.Set(null, null);
```

Comment:
All the existing tests check the issues collection, the new property is standalone so I just passed null everywhere and added new dedicated tests.